zips/zip-0000.rst

```diff
@@ -220,7 +221,12 @@ Whenever the ZIP Editors change, the new ZIP Editors SHOULD:
 * update the <zips@z.cash> email alias, and
 * invite new Editors to the Zcash Community Forum, and the #zips channel on Discord.
 
-If it has been at least 12 months since the last ZIP Editor change, the ZIP Editors SHOULD:
+Whenever it has been at least 12 months since either:
+
+* the last ZIP Editor change; or
+* this ZIP was last reviewed,
+
+the ZIP Editors SHOULD:
 
 * review this ZIP to make sure it reflects current practice.
```

Comment:
```suggestion
Whenever it has been at least 12 months since this ZIP was last reviewed
(whether or not that was a review in response to a ZIP Editor change), the
ZIP Editors SHOULD review this ZIP again to make sure that it reflects
current practice.
```